src/plugin/file/kml/kml.js

```diff
@@ -217,6 +217,42 @@ plugin.file.kml.readStyle = function(node, objectStack) {
 };
 
 
+/**
+ * Parse text field from BalloonStyle tag.
+ * @param {Node} node Node.
+ * @return {string|*}
+ * @private
+ */
+plugin.file.kml.readBalloonText_ = function(node) {
```

Comment:
This function is identical to `ol.format.XSD.readString`. Please use that instead.